src/Microsoft.ML.Dnn/DnnUtils.cs

```diff
@@ -92,11 +92,10 @@ internal static Session LoadTFSession(IExceptionContext ectx, byte[] modelBytes,
 
         internal static Graph LoadMetaGraph(string path)
         {
-            return tf_with(tf.Graph().as_default(), graph =>
-            {
-                tf.train.import_meta_graph(path);
-                return graph;
-            });
+            var graph = new Graph();
+            graph = graph.as_default();
+            tf.train.import_meta_graph(path);
```

Comment:
>tf.train.import_meta_graph(path); [](start = 12, length = 33)

It seems like there is no connection between `graph` and `tf`, does this line do anything to `graph`?

Reply:
This creates a new graph and the 'as_default()' sets this as the tf default graph [https://github.com/SciSharp/TensorFlow.NET/blob/41432600c8263cf972b13930c31479ffb412fa65/src/TensorFlowNET.Core/Graphs/Graph.cs#L134](https://github.com/SciSharp/TensorFlow.NET/blob/41432600c8263cf972b13930c31479ffb412fa65/src/TensorFlowNET.Core/Graphs/Graph.cs#L134). The 'graph' is where the meta graph is loaded and returned. 

---
In reply to: [324450631](https://github.com/dotnet/machinelearning/pull/4205#discussion_r324450631) [](ancestors = 324450631)